spec/openapi.yml

```diff
@@ -229,7 +229,8 @@ paths:
   '/{authenticator}/{service_id}/{account}/login':
     $ref: 'authentication.yaml#/components/paths/ServiceLogin'
 
-# ========== STATUS ===================
+  '/authn-gcp/{account}/authenticate':
+    $ref: 'authentication.yaml#/components/paths/GCPAuthenticate'
```

Comment:
I think you accidentally removed the `STATUS` header, should be below the new entry.